ehrapy/anndata/anndata_ext.py

```diff
@@ -154,63 +154,135 @@ def anndata_to_df(
     return df
 
 
-def move_to_obs(adata: AnnData, to_obs: list[str] | str, copy: bool = False) -> AnnData:
-    """Move features from X to obs inplace. Note that columns containing boolean values (either 0/1 or True(true)/False(false))
-    will be stored as boolean columns whereas the other non numerical columns will be stored as category.
+def move_to_obs(adata: AnnData, to_obs: list[str] | str, copy_obs: bool = False, copy: bool = False) -> AnnData:
+    """Move inplace or copy features from X to obs. Note that columns containing boolean values (either 0/1 or True(
+    true)/False(false)) will be stored as boolean columns whereas the other non numerical columns will be stored as
+    category.
 
     Args:
         adata: The AnnData object
         to_obs: The columns to move to obs
+        copy_obs: The values are copied to obs (and therefore kept in X) instead of moved completely
         copy: Whether to return a copy or not
```

Comment:
Maybe replace with something like: `copy: Whether to return a copy of the original AnnData object or not.`. So it's clear, we're talking about `AnnData` objects here; much copying going on here so might be useful.